content/test/integration/Server.spec.ts

```diff
@@ -42,11 +43,13 @@ describe('Integration - Server', function () {
       .registerBean(Bean.REPOSITORY, MockedRepository.build())
       .registerBean(Bean.SERVICE, service)
       .registerBean(Bean.SYNCHRONIZATION_MANAGER, new MockedSynchronizationManager())
+      .registerBean(Bean.DENYLIST, new DeactivatedDenylist())
```

Comment:
Why are you adding this and the config? I think that the config be enough on it's own, but why was it necessary in the first place?

Reply:
Because this emulates the behaviour under the `DeploymentFactory`, I needed to add this for the test cases to work.

Reply:
We will use the activated denylist, and remove the config.

Also, we will enable the denylist by default